nvtabular/io/parquet.py

```diff
@@ -693,7 +700,8 @@ def _write_pq_metadata_file(md_list, fs, path):
         metadata_path = fs.sep.join([path, "_metadata"])
         _meta = cudf.io.merge_parquet_filemetadata(md_list) if len(md_list) > 1 else md_list[0]
         with fs.open(metadata_path, "wb") as fil:
-            _meta.tofile(fil)
+            for m in _meta:
+                fil.write(m)
```

Comment:
```suggestion
            fil.write(bytes(_meta))
```
I think we should write the entire metadata blob at once (rather than looping over every byte).